packages/amplify-cli/src/utils/team-provider-migrate.ts

```diff
@@ -0,0 +1,18 @@
+import { Context } from '../domain/context';
+import { stateManager, pathManager } from 'amplify-cli-core';
+import _ from 'lodash';
+import { externalAuthEnable } from 'amplify-category-auth';
+const message = `Amplify auth will be modified to mangage secrets from deployment-secrets.json. Would you like to proceed?`;
```

Comment:
@renebrandel I will need your input for the migration message

Reply:
Does the customer really have a choice? What happens if they say no? Should this decision be behind a FF instead of a prompt?

Rene should have the final say in the experience here, but as a customer I think I would be confused about how to answer this question